docs/guide/search/search_criteria_reference.md

```diff
@@ -60,6 +57,14 @@ Due to this storage limitation, searching content using the Country Field Type o
 |[UserMetadata](criteria_reference/usermetadata_criterion.md)|The creator or modifier of a Content item|Content and Location Search; Filtering|
 |[Visibility](criteria_reference/visibility_criterion.md)|Whether the Content item is visible or not|Content and Location Search; Filtering|
 
+### Product search
```

Comment:
I guess there is a specific reason why product catalog criteria are treated in a special way?

Reply:
Yes, they are used for different searches.